src/System.Net.Http/tests/FunctionalTests/HttpClientHandlerTest.cs

```diff
@@ -162,6 +162,33 @@ public void MaxRequestContentBufferSize_Set_ThrowsPlatformNotSupportedException(
             }
         }
 
+        [Theory]
+        [MemberData(nameof(GetAsync_IPv6Uri_Success_MemberData))]
+        public async Task GetAsync_IPv6Uri_Success(IPAddress address)
+        {
+            using (var client = new HttpClient())
+            {
+                var options = new LoopbackServer.Options { Address = address };
+                await LoopbackServer.CreateServerAsync(async (server, url) =>
+                {
+                    await TestHelper.WhenAllCompletedOrAnyFailed(
```

Comment:
Is this the new suggested pattern for writing tests using the loopback server, i.e. this `TestHelper.WhenAllCompletedOrAny` method?

I'm planning to convert more of the current Http tests to use the loopback server (to avoid timeout latency issues with the Azure endpoint) and so I want to know the best pattern to use.